ecli/dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:22.04
+
+ENV UBUNTU_SOURCE /etc/apt
+
+WORKDIR /root
+
+RUN apt-get update -y && \
+	apt-get install -y --no-install-recommends libelf1 libelf-dev \
+		zlib1g-dev clang llvm libclang-13-dev make cmake git  wget tar && \
+	apt-get clean && \
+	curl https://sh.rustup.rs -sSf | sh -s	&& \
+	rm -rf /var/lib/apt/lists/*	&& \
+	apt-get update	&& \
+	apt-get install -y ca-certificates	&& \
+	update-ca-certificates
+
+RUN git clone --recurse-submodules https://github.com/eunomia-bpf/eunomia-bpf.git	&& \
```

Comment:
Why not using the copy COMMAND? see https://github.com/eunomia-bpf/eunomia-bpf/blob/master/compiler/dockerfile#L11-L11